api_app/views.py

```diff
@@ -82,7 +85,19 @@ def _multi_analysis_request(
     else:
         # fire celery task
         for index, job in enumerate(jobs):
-            runtime_configuration = runtime_configurations[index]
+            runtime_configuration = {}
+
+            for analyzer in job.analyzers_to_execute:
+                # Appending custom config to runtime configuration
+                config = CustomConfig.get_as_dict(
+                    user, CustomConfig.Type.ANALYZER, name=analyzer
+                ).get(analyzer, {})
+                if analyzer in runtime_configurations[index]:
+                    config = config | runtime_configurations[index][analyzer]
+                if config:
+                    runtime_configuration[analyzer] = config
+            logger.debug(runtime_configuration, type(job.analyzers_to_execute[0]))
```

Comment:
cool but there is no normal text. This is of critical importance when you need to search through your logs for the all the logs generated by this line. It is enough to write few words with this